modules/machine-autoscaler-cr.adoc

```diff
@@ -24,9 +24,16 @@ spec:
     kind: MachineSet <5>
     name: worker-us-east-1a <6>
 ----
-<1> Specify the machine autoscaler name. To make it easier to identify which machine set this machine autoscaler scales, specify or include the name of the machine set to scale. The machine set name takes the following form: `<clusterid>-<machineset>-<aws-region-az>`
-<2> Specify the minimum number machines of the specified type that must remain in the specified zone after the cluster autoscaler initiates cluster scaling. If running in AWS, GCP, Azure, or {rh-openstack}, this value can be set to `0`. For other providers, do not set this value to `0`.
+<1> Specify the machine autoscaler name. To make it easier to identify which machine set this machine autoscaler scales, specify or include the name of the machine set to scale. The machine set name takes the following form: `<clusterid>-<machineset>-<aws-region-az>`.
+<2> Specify the minimum number machines of the specified type that must remain in the specified zone after the cluster autoscaler initiates cluster scaling. If running in AWS, GCP, Azure, {rh-openstack}, or vSphere, this value can be set to `0`. For other providers, do not set this value to `0`.
 <3> Specify the maximum number machines of the specified type that the cluster autoscaler can deploy in the specified AWS zone after it initiates cluster scaling. Ensure that the `maxNodesTotal` value in the `ClusterAutoscaler` resource definition is large enough to allow the machine autoscaler to deploy this number of machines.
 <4> In this section, provide values that describe the existing machine set to scale.
 <5> The `kind` parameter value is always `MachineSet`.
 <6> The `name` value must match the name of an existing machine set, as shown in the `metadata.name` parameter value.
+
+If your machines are running in AWS, GCP, Azure, {rh-openstack}, or vSphere, you can set the `spec.minReplicas` value to `0`. You can save on costs by setting this value to zero for use cases such as running expensive or limited-usage hardware used for specialized workloads, or scaling a machine set with extra large machines. The machine autoscaler scales the machine set down to zero when the machines are not in use.
+
+[IMPORTANT]
+====
+Do not set the `spec.minReplicas` value to `0` for the three compute plane machine sets that are created during the {product-title} installation process for an installer provisioned infrastructure.
```

Comment:
Should this be a warning/caution note? What is the consequence of performing this action?

Not sure if the order of the previous line -> this is awkward. I'm essentially told to set this value to 0 to save money (who doesn't want that?), and then I'm warned not to--at least within a particular context. 

My emotional journey: :chart_with_upwards_trend: woohoo! -> :chart_with_downwards_trend: oh

Reply:
yeah, this is exactly the grey area around scaling to zero. there are several great reasons for users to exercise this functionality, and cost/resource saving is primary amongst them. BUT, if a user tries to scale down their initial compute plane, they will send the clusters into a very unhappy state. i _think_ we have sufficient gates in place in the code to prevent the cluster from scaling those nodes down, but it is possible. imo, it's in the best interest of users that they understand how to use this properly, and when not to use it.

perhaps escalating to use warning/caution is good in this case.